includes/core-changes/windowsforms/cellformatting-event-not-raised.md

```diff
@@ -0,0 +1,33 @@
+### CellFormatting event not raised if tooltip is shown
+
+To meet accessibility standards, a <xref:System.Windows.Forms.DataGridView> now shows a cell's text and errors as tooltips when navigated by both mouse and keyboard. If a tooltip is shown, the <xref:System.Windows.Forms.DataGridView.CellFormatting?displayProperty=nameWithType> event is not raised.
+
+#### Change description
+
+Previous to .NET Core 3.1, a <xref:System.Windows.Forms.DataGridView> would only show a cell's tooltips and error text when navigated by mouse. If the user edited a cell that had the <xref:System.Windows.Forms.DataGridView.ShowCellToolTips%2A> property set to `true`, the <xref:System.Windows.Forms.DataGridView.CellFormatting> event was raised whenever other cells that did not have <xref:System.Windows.Forms.DataGridView.ShowCellToolTips%2A> set were hovered over before the edit was committed.
```

Comment:
I'm finding the mix of "navigating," "hovering," and "editing" here and in the next paragraph hard to understand.  I don't know if it will be clear enough to people who are familiar with the API.